tests/tests_app/utilities/test_cli_helpers.py

```diff
@@ -111,3 +118,51 @@ def test_get_newer_version(mock_requests, releases, current_version, newer_versi
 
     _get_newer_version.cache_clear()
     assert _get_newer_version() == newer_version
+
+
+@patch("lightning.app.utilities.cli_helpers._redirect_command")
+def test_check_environment_and_redirect(mock_redirect_command, tmpdir):
+    original_path = os.environ["PATH"]
+
+    # Ensure that the test fails if it tries to redirect
+    mock_redirect_command.side_effect = RuntimeError
+
+    try:
```

Comment:
you could use monkeypatch to make the test a bit cleaner (without try-except)

Reply:
Cool! Yeah that's much better 😃